frontend/public/index.html

```diff
@@ -29,12 +29,27 @@
     </noscript>
     <div>
       <div id="app">
-        <div class="overlay">
-          <div class="loadingWrapper">
-            <div class="loader">Loading...</div>
-            <h2>Loading</h2>
-          </div>
-        </div>
+        <div class="quote-wrapper"><q class="quote"></q></div>
+        <script>
+        function getRandomQuote(quotes) {
+          return quotes[Math.floor(Math.random() * quotes.length)];
+        }
+
+        function showQuote(quote) {
+          document.querySelector(".quote").appendChild(document.createTextNode(quote))
+        }
+        const quotes = localStorage.getItem("quotes");
+        if (quotes) {
+          showQuote(getRandomQuote(JSON.parse(quotes)));
+        } else {
+          let defaultQuote = localStorage.getItem("defaultQuote");
+          if (!defaultQuote) {
+            localStorage.setItem("defaultQuote", "I am closer to you than the air you breathe.");
+            defaultQuote = "I am closer to you than the air you breathe.";
+          }
+          showQuote(defaultQuote);
+        }
```

Comment:
I think it's over complicated.

- No need to save the default quote into the storage.
- The text of the default quote should not be mentioned twice in the source code.
- You can pick randomly from one default quote to streamline the code flow.
- You even can save the default quote to the storage if it's empty, later the default quote will be overwritten from the database.